config/censor.js

```diff
@@ -7,12 +7,49 @@ var util = require('util');
 // Express middleware to check for words that should not be allowed
 var censor = {
   isNaughty: function(req, res, next) {
-    var reqString = util.inspect(req).toLowerCase();
-    for (var i = 0; i < length; i++) {
-      if (reqString.indexOf(naughtylist[i]) > -1) {
-        return res.status(422).send('Inappropriate content detected.');
+
+    // Check if a word is in the naughty list
+    var isNaughtyWord = function(word) {
+      for (var i = 0; i < length; i++) {
+        if (word === naughtylist[i]) {
+          return true;
+        }
+      }
+      return false;
+    };
+
+    // Examine req.params
```

Comment:
Looks good, but we do need to be careful about for ... in.
See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)'s section on hasOwnProperty for an explanation of why. It's totally correct in this case, just wanted to alert you to the danger's of for ... in.